simSPI/cryoemio.py

```diff
@@ -0,0 +1,392 @@
+import numpy as np
+import h5py
+import os
+import mrcfile
+import CifFile
+from CifFile import ReadCif
+
+
+## cryoSPARC related ##
```

Comment:
No leftover comments, put that in the docstring. --> please add docstrings (see comment on other PR)